src/components/views/rooms/EventTile.tsx

```diff
@@ -1428,24 +1410,27 @@ export class UnwrappedEventTile extends React.Component<EventTileProps, IState>
                             </div>
                             {this.renderThreadPanelSummary()}
                         </div>
-                        <Toolbar className="mx_MessageActionBar" aria-label={_t("Message Actions")} aria-live="off">
-                            <RovingAccessibleTooltipButton
-                                className="mx_MessageActionBar_iconButton"
-                                onClick={this.viewInRoom}
-                                title={_t("View in room")}
-                                key="view_in_room"
-                            >
-                                <ViewInRoomIcon />
-                            </RovingAccessibleTooltipButton>
-                            <RovingAccessibleTooltipButton
-                                className="mx_MessageActionBar_iconButton"
-                                onClick={this.copyLinkToThread}
-                                title={_t("Copy link to thread")}
-                                key="copy_link_to_thread"
-                            >
-                                <LinkIcon />
-                            </RovingAccessibleTooltipButton>
-                        </Toolbar>
+                        {this.context.timelineRenderingType === TimelineRenderingType.ThreadsList && (
+                            <Toolbar className="mx_MessageActionBar" aria-label={_t("Message Actions")} aria-live="off">
```

Comment:
Same comment than before, can we move it to a dedicated component ?

Reply:
Made sense to create a toolbar item here!